### DISCUSSION TITLE
服务化部署，image参数限制1M，如何配置此参数

### DISCUSSION THREAD

**Discussion by donghao85** [PaddlePaddle/PaddleX#2635] [General]
服务化部署，通用ocr产线，根据文档，将图片进行base64编码通过image参数上传，但默认限制1M，如何提升这个参数的大小限制？

**Comment by Bobholamovic** [comment]
paddlex的服务化部署方案本身并未对图像大小施加限制，这个1M的限制可能是来自客户端，或者你所使用的某个网关。此外，图像体积过大的情况下，一般不建议使用base64上传，因为base64编码会让请求体的体积进一步膨胀，可以考虑传入URL。
